### PR TITLE
[EAGLE-860] TopologyDataExtractor can't extract right rack info

### PR DESCRIPTION
**Pull request by r7raul1984**
- Use clusternode API to get rack info.

https://issues.apache.org/jira/browse/EAGLE-860

### CODE REVIEW

File: eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/resolver/model/Node.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.apache.eagle.topology.resolver.model;
+
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
```
Comment (haoch) [inline eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/resolver/model/Node.java]: Please use `faster-xml jackson`

File: eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/resolver/model/NodeInfo.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.apache.eagle.topology.resolver.model;
+
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
```
Comment (haoch) [inline eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/resolver/model/NodeInfo.java]: Please use `faster-xml jackson`

Comment (r7raul1984) [inline eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/resolver/model/NodeInfo.java]: OK

**Comment by haoch** [comment]
LGTM

**Pull request closed without merging**